### ISSUE TITLE
metadaten - Kategorie-filter werden ignoriert

### ISSUE THREAD

**Issue by staabm** [FriendsOfREDAXO/minibar#17]
https://github.com/redaxo/redaxo/issues/2387

**Comment by eaCe** [comment]
Was wenn keine Kategorie vorhanden? z.B. Startartikel. @skerbis

**Comment by skerbis** [comment]
Stimmt! Ist dies ein Bug im Core? @staabm @gharlan ... Für root kann man garnicht filtern ....

**Comment by fietstouring** [comment]
Kurze Frage: die minibar zeigt aktuell bei mir die Metainformationen nur im Backend an, nicht im Frontend. Ist das nur bei mir so? 
Redaxo 5.8

**Comment by eaCe** [comment]
Das ist gewollt. Die Metainfos werden nur im Backend geladen.
